README.md

```diff
@@ -1,4 +1,4 @@
-# gtk-test [![Build Status](https://travis-ci.org/gtk-rs/gtk-test.png?branch=master)](https://travis-ci.org/gtk-rs/gtk-test) [![Build status](https://ci.appveyor.com/api/projects/status/h72xnw2ghjpy2m9y/branch/master?svg=true)](https://ci.appveyor.com/project/GuillaumeGomez/gtk-test/branch/master)
+# gtk-test [![Build Status](https://github.com/gtk-rs/gtk-test/actions/workflows/linux_ci.yml/badge.svg)](https://github.com/gtk-rs/gtk-test/actions/workflows/linux_ci.yml) [![Build Status](https://github.com/gtk-rs/gtk-test/actions/workflows/mac_ci.yml/badge.svg)](https://github.com/gtk-rs/gtk-test/actions/workflows/mac_ci.yml) [![Build status](https://ci.appveyor.com/api/projects/status/h72xnw2ghjpy2m9y/branch/master?svg=true)](https://ci.appveyor.com/project/GuillaumeGomez/gtk-test/branch/master)
```

Comment:
You can remove the badges I think. Also, just realized that we still had badge for appveyor. Please also remove all appveyor-related things while you're at it. :)

Reply:
Okay, all badges and appveyor-related items have been removed.